include/smack/SmackRep.h

```diff
@@ -184,8 +186,10 @@ class SmackRep {
   virtual vector<string> getModifies();
   unsigned numElements(const llvm::Constant* v);
   void addInit(unsigned region, const llvm::Value* addr, const llvm::Constant* val);
+  void addInitFunc(const Stmt* s);
```

Comment:
I was just saying that the signature of addInitFunc should be:
void addInitFunc(const llvm::Function* func);
Then this Stmt::call(naming.get(*func)) should be encapsulated in this function, as opposed to being done in SmackModuleGenerator.